.azuredevops/components/rocminfo.yml

```diff
@@ -5,14 +5,18 @@ parameters:
 - name: checkoutRef
   type: string
   default: ''
+- name: rocmDependencies
+  type: object
+  default:
+    - ROCR-Runtime
 
 jobs:
 - job: rocminfo
   variables:
   - group: common
   - template: /.azuredevops/variables-global.yml
   pool:
-    vmImage: ${{ variables.LOW_END_BUILD_POOL }}
+    vmImage: ${{ variables.BASE_BUILD_POOL }}
```

Comment:
I think BASE_BUILD_POOL is a local machine. Wonder if that's desired

Reply:
BASE_BUILD_POOL is not a local machine. It is the ubuntu-22.04, Microsoft-hosted pool. This is desired. ON_PREM_BUILD_POOL are local machines.